src/eddymotion/model/tests/test_dmri_utils.py

```diff
@@ -0,0 +1,620 @@
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                        3000,
+                    ]
+                ),
+            ],
+        ),
+        (
+            "ds004737_dsi",
+            "DSI",
+            [
+                np.asarray([5, 5, 5, 5, 5, 5, 5, 5, 5]),
+                np.asarray([995, 995, 800, 800, 995, 995, 795, 995]),
+                np.asarray([1195, 1195, 1195, 1195, 1000, 1195, 1195, 1000]),
+                np.asarray([1595, 1595, 1595, 1600.0]),
+                np.asarray(
+                    [
+                        1800,
+                        1795,
+                        1795,
+                        1790,
+                        1995,
+                        1800,
+                        1795,
+                        1990,
+                        1990,
+                        1795,
+                        1990,
+                        1795,
+                        1795,
+                        1995,
+                    ]
+                ),
+                np.asarray([2190, 2195, 2190, 2195, 2000, 2000, 2000, 2195, 2195, 2190]),
+                np.asarray([2590, 2595, 2600, 2395, 2595, 2600, 2395]),
+                np.array([2795, 2790, 2795, 2795, 2790, 2795, 2795, 2790, 2795]),
+                np.array([3590, 3395, 3595, 3595, 3395, 3395, 3400]),
+                np.array([3790, 3790]),
+                np.array([4195, 4195]),
+                np.array([4390, 4395, 4390]),
+                np.array(
+                    [
+                        4790,
+                        4990,
+                        4990,
+                        5000,
+                        5000,
+                        4990,
+                        4795,
+                        4985,
+                        5000,
+                        4795,
+                        5000,
+                        4990,
+                        4990,
+                        4790,
+                        5000,
+                        4990,
+                        4795,
+                        4795,
+                        4990,
+                        5000,
+                        4990,
+                    ]
+                ),
+            ],
+        ),
+    ],
+)
+def test_find_shelling_scheme_files(dwi_btable, exp_scheme, exp_bval_groups):
+    bvals = np.loadtxt(_datadir / f"{dwi_btable}.bval")
+
+    obt_scheme, obt_bval_groups = find_shelling_scheme(bvals)
```

Comment:
```suggestion
    obt_scheme, obt_bval_groups, obt_bval_estimates = find_shelling_scheme(bvals)
```

Reply:
Left behind in my previous commit. Added in d221f03.